libs/src/api/Track.ts

```diff
@@ -343,12 +344,19 @@ export class Track extends Base {
    * @param metadata json of the track metadata with all fields, missing fields will error
    * @param onProgress callback fired with (loaded, total) on byte upload progress
    */
-  async uploadTrack(
-    trackFile: File,
-    coverArtFile: File,
-    metadata: TrackMetadata,
+  async uploadTrack({
```

Comment:
is there a way we are tracking these api changes for when we update in audius-client?

Reply:
Yes, this seemed scary to me in the case that we revert the libs version and not doing it atomically with the client change. I have the client changes out in PR already too. will try to merge together with libs update